index.html

```diff
@@ -332,6 +332,33 @@ <h3 class="lets-connect">LET’S CONNECT</h3>
       </div>
 
     </section>
+    <section class="contact">
+      <div class="contact-wrap">
```

Comment:
- [ ] Please use a `footer` tag instead of a `section` tag because the contact section is the footer for your portfolio website :+1: Also, it would be better for SEO purposes. :blush: